artifacts/scripts/construct.sh

```diff
@@ -65,7 +65,7 @@ SOURCE_REPO_NAME="${9}"
 shift 9
 
 # base package name (eg. k8s.io)
-BASE_PACKAGE="${1-k8s.io}"
+BASE_PACKAGE="${1:-k8s.io}"
```

Comment:
@SataQiu Thanks! 

Just to be clear, this is not a syntax error. :)

`${foo:bar}` expands to `bar` when foo is unset, but `""` when foo is null, while `${foo:-bar}` expands to `bar` if foo is unset or null.

Reply:
@nikhita Thank you for your patient explanation. :)